resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -62,15 +63,20 @@ export default {
       showWizardTemplates: false,
       showCardProcesses: false,
       category: null,
+      numCategories: 15,
     };
   },
   mounted() {
     this.getCategories();
   },
   methods: {
+    addCategories() {
+      this.numCategories += 15;
+      this.getCategories();
+    },
     getCategories() {
       ProcessMaker.apiClient
-        .get("process_categories")
+        .get(`process_categories?per_page=${this.numCategories}`)
```

Comment:
improve this call, 
In the array listCategories has to be added in increments of 15.